styx-scheduler-service/src/test/java/com/spotify/styx/StateInitializingTriggerTest.java

```diff
@@ -123,19 +140,37 @@ public void shouldInjectTriggerExecutionEventWithBackfillTrigger() throws Except
   }
 
   @Test
-  public void shouldDoNothingIfDockerImageMissing() {
-    final WorkflowConfiguration configuration =
+  @Parameters(method = "noDockerImageConfigurations")
+  public void shouldDoNothingIfDockerImageMissing(WorkflowConfiguration configuration) {
```

Comment:
What is the benefit of having this method Parameterized instead of having a static generation of the configuration, or defining it in the beginning of the method?

Reply:
The test is running for multiple configurations

Reply:
Ah right. Didn't notice it was 2 configurations. cool!